third-party/rsutils/include/rsutils/json.h

```diff
@@ -11,6 +11,9 @@ namespace rsutils {
 namespace json {
 
 
+extern nlohmann::json const null_json;
```

Comment:
1. Why is `extern` variable better than `static const`?

2. Can we define the variable here and avoid the need for the cpp file?
`extern nlohmann::json const null_json = {};`

Reply:
You can't `extern` and define the value on the same line: they contradict.
`extern` means it's defined somewhere else.
`static` means it's local to the current compilation unit, so repeated in each one if it's in a header.
I wanted an object I could return the reference of, and only one.